### PR TITLE
3685 Updated advanced search docs for explaining parties search and filtering

### PR DESCRIPTION
**Pull request by albertisfu**
I've added a note about filtering and searching for party and attorney to the search documentation:

Following your suggestion, I've included some examples to explain how to include empty filings in search results when using a party and/or attorney field.

Screenshot 2024-01-29 at 13 05 26

Let me know what you think.

Fixes: #3685

### CODE REVIEW

**Comment by mlissner** [comment]
OK, I guess I'm a bit confused, and I'm sorry because I know you've explained this a dozen times, but is the issue only when you combine party/atty fields with non-docket fields in cases with no filings? Because if so, I don't understand how that's an issue: If a case has no filings, how would a non-docket field do anything anyway?

**Comment by albertisfu** [comment]
Yeah, checked the test I added in https://github.com/freelawproject/courtlistener/issues/3685 along with some alternative query structures, and I think you're correct. We should be able to use parent filters combined with party fields, and a child field filter shouldn't affect the matching of a docket with no filings when using parties. Therefore, I'll apply the required fixes in #3685, and this comment might change or not be required anymore.

**Comment by mlissner** [comment]
OK, that's definitely getting better, but I'm not sure I even know the difference between fielded searches and filters. Too technical. 

Maybe say that these can't be combined in the main query box, and instead need to be done via the sidebar?

**Comment by albertisfu** [comment]
Great, I have updated the language to be less technical. Hope it's better.

Screenshot 2024-01-30 at 10 30 05

File: cl/simple_pages/templates/includes/available_fields_tabs.html [hunk 1]
```diff
@@ -29,6 +29,21 @@
...
+          <ul>
+            <li><code>party:(Party Name) AND attorney:(Attorney) AND description:(some description)</code></li>
+          </ul>
+          <p>Will not return any results. To achieve the desired results, utilize filters available in the sidebar:</p>
+          <ul>
+            <li><code>Document Description: "some description"</code></li>
+            <li><code>Party Name: "The party name"</code></li>
+            <li><code>Attorney  Name: "The attorney name"</code></li>
+          </ul>
+          <p>This approach will retrieve filings that match the specified document description and belong to dockets matching the provided Party and Attorney names.</p>
```
Comment (mlissner) [inline cl/simple_pages/templates/includes/available_fields_tabs.html:45]:
```suggestion
          <p>This approach will retrieve filings that match the specified document description in dockets with the specified parties and attorneys.</p>
```

**Review by mlissner — APPROVED** [review]
I made a little tweak, but, yes, I think this will work. Documentation is hard!

**Pull request merged**